bin/all_sky_search/pycbc_add_statmap

```diff
@@ -68,6 +74,23 @@ if args.max_hierarchical_removal and injection_style:
     raise NotImplementedError("Hierarchical background removal doesn't make "
                               "sense for injections.")
 
+# Check that the user chose inclusive or exclusive background to perform
+# hierarchical removals of foreground triggers against.
+if args.max_hierarchical_removal == 0:
+    if args.hierarchical_removal_against != 'none':
+        parser.error("User Error: 0 maximum hierarchical removals chosen but "
+                     "option for --hierarchical-removal-against was given. "
+                     "These are conflicting options. Use with --help for more "
+                     "information.")
+else :
+    is_bkg_inc = (args.hierarchical_removal_against == 'inclusive')
```

Comment:
I think you could save a couple of variables and a few lines by doing the check on 
`if args.hierarchical_removal_against is 'none'` and later also switching on the args attribute .. these two Boolean variables are basically not used for anything beyond repackaging the option value

Reply:
i.e. if you can get by without adding variables that more or less reproduce the options, do so

Reply:
That's fair - this logic was copied over from other files which use HR, so I've edited it in the coinc_statmap file as well

Reply:
Looks OK.  (Coinc statmap HR has very chatty comments doesn't it!)